From 7cb08c26be3eab9518c5e4c6f513e7ce6f38855f Mon Sep 17 00:00:00 2001
From: Ruthwar Ambeer <rambeer@codeaurora.org>
Date: Wed, 8 Aug 2018 17:04:51 +0530
Subject: [PATCH] Disable IOemHook implemenation in rild.

By default IOemHook hal is enabled and its running
in the rild process. So disabling the hal to avoid
any VTS failures as it is deprecated.

Change-Id: If63741829d83679611871bfd2ab600fdba6673c5
CRs-Fixed: 2293852
---
 libril/Android.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libril/Android.mk b/libril/Android.mk
index cde192f..d5c0e2e 100644
--- a/libril/Android.mk
+++ b/libril/Android.mk
@@ -37,7 +37,7 @@ ifeq ($(SIM_COUNT), 2)
     LOCAL_CFLAGS += -DANDROID_SIM_COUNT_2
 endif
 
-ifneq ($(DISABLE_RILD_OEM_HOOK),)
+ifneq ($(DISABLE_RILD_OEM_HOOK), false)
     LOCAL_CFLAGS += -DOEM_HOOK_DISABLED
 endif
 
-- 
2.17.1

